src/elements/ts-settings/ts-settings.html

```diff
@@ -293,6 +293,19 @@
             this.set(path, value);
             this.closeSettingModal();
             this.saveSettings();
+            if (name === "serversuite") {
+                App.configurator.setUserSetting("dataserver", value.dataserver);
+                App.configurator.setUserSetting("mediaserver", value.mediaserver);
+                this.set('settings', App.configurator.refreshUserSetting());
+                this.saveSettings();
+                // Lgout user.
+                this.fire('iron-signal', {name: 'updateuser', data: {}});
+                App.keyManager.destroyKeys();
+                // Notify user that the app needs to be restarted.
+                // TODO This bypasses Polymer to call the ts-dashboard.showMessage() function.  Is there a better way?
```

Comment:
Is this todo resolved?

Reply:
No.  I couldn't find any other way to pop up the dialog.  Suggestions welcome.

Reply:
Nope just saw the todo wanted to make sure you had a chance to look at it.